news/_posts/2021-03-12-ABCD-ReproNim-ends.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: news
+title: The ABCD-ReproNim course wraps up year one
+author: Taylor Salo
+category: news
+tags: []
+image: /assets/images/news/abcd_repronim.png
+excerpt: The virtual ABCD-ReproNim course has successfully completed its first year!
+---
+
+The [ABCD-ReproNim Course](https://www.abcd-repronim.org), a virtual course providing training on reproducible analysis of the ABCD dataset,
+has completed its final session, a week-long period of group projects, on March 12.
```

Comment:
@jessicabartley would you mind adding some stuff here? You can just suggest it in this thread.

Reply:
Done! lmk if we want to add anything else!